actions.php

```diff
@@ -9,16 +9,25 @@
   // Get the collection.
   $collection = $app->module('collections')->collection($name);
 
-  // Get the collection fields.
+  // Get the collection fields' definitions.
   $fields = [];
   foreach ($collection['fields'] as $field) {
     $fields[$field['name']] = $field;
   }
 
   foreach ($data as $idx => $entry) {
-    foreach ($entry as $fieldName => $values) {
-      // Check for the field type and inspect for presence of styles.
-      switch ($field['type']) {
+    foreach ($entry as $fieldName => $values) {    
+      
+      // Check if current field in entry is defined by user, we don't want '_id' etc. to proceed.
+      if(!array_key_exists($fieldName, $fields)) {
```

Comment:
please add a blank space between if and (
also will be better to use isset instead of array_key_exists, we want that the entry exists and is not null